grammars/go.cson

```diff
@@ -162,19 +169,37 @@
         "match": "(\\+|&|\\+=|&=|&&|==|!=|\\-|\\||\\-=|\\|=|\\|\\||<|<=|\\*|\\^|\\*=|\\^=|<\\-|>|>=|/|<<|/=|<<=|\\+\\+|=|:=|;|%|>>|%=|>>=|\\-\\-|!|\\.\\.\\.|&\\^|&\\^=)"
       }
       {
-        "name": "keyword.operator.bracket.go",
-        "match": "(\\(|\\)|\\[|\\]|\\{|\\})"
+        'match': '\\{|\\}'
+        'name': 'meta.brace.curly.go'
       }
       {
-        "name": "keyword.operator.punctuation.go",
-        "match": "(\\.|,|:)"
+        'match': '\\(|\\)'
+        'name': 'meta.brace.round.go'
+      }
+      {
+        'match': '\\[|\\]'
+        'name': 'meta.brace.square.go'
       }
     ]
   'variables':
     'patterns': [
       {
-        'name': 'variable.go'
-        'match': '\\b[\\w_][\\w\\d_]*\\b',
+        'match': '\\b[\\w_][\\w\\d_]*\\b'
```

Comment:
This could be added back, if you prefer. We'd have to be careful around type declarations, etc. since a type name isn't a variable.

```go
type User struct {
  name string
  yob  int
}